Documentation/upgrade.md

```diff
@@ -238,16 +291,36 @@ Each of these replica sets will need to be edited to update the image version, t
 
 In this example, we are going to walk through the case where `useAllNodes: true` was set in the cluster spec, so there will be a single daemon set managing all the OSD pods.
 
-Let's edit the image version of the pod template in the daemon set:
-```bash
-kubectl -n rook set image daemonset/rook-ceph-osd rook-ceph-osd=rook/rook:v0.5.1
+In addition to updating the version of the container, we will need to add a new environment variable for the `uid` of the cluster. First, we will look up the value of the cluster `uid` that is required for the new variable.
+```
+kubectl -n rook get cluster rook -o jsonpath='{.metadata.uid}'
```

Comment:
maybe show the example output here that gives the user a sense of what a uid is supposed to look like